client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -84,8 +128,25 @@ class RequiredPluginsInstallView extends Component {
 		};
 	};
 
+	doTransferStatusPolling = () => {
+		const { automatedTransferStatus, siteId } = this.props;
+
+		const { COMPLETE } = transferStates;
+
+		if ( automatedTransferStatus === COMPLETE ) {
+			this.setState( {
+				engineState: 'INITIALIZING',
+				workingOn: '',
+				stepIndex: 3,
+				numTotalSteps: 6,
```

Comment:
Same comment above explaining why 3, why 6, and what changes might warrant a change here. Even better: constants at the top of the file for both.